docs/performance/tune-performance.md

```diff
@@ -170,6 +170,7 @@ Currently, there are no special provisions to employ mimalloc on Linux. It is re
 ### Thread management
 
 * Use the appropriate ORT API to set intra and inter op num threads. Inter op num threads is only used when parallel execution is enabled.
+* By default, both of these values are set to half of the number of threads supported by the current implementation.
```

Comment:
Given that this doc is intended to be consumed by developers, we can just say ```std::max(1, static_cast<int>(std::thread::hardware_concurrency() / 2));``` to avoid any ambiguity around "number of threads supported".

Reply:
My only thought about that was that this is the Python API and if the developer is not familiar with the CPP standard libraries then it might put them off

Reply:
Yeah, in that case may be link to the doc [here](https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency) or say ```half the number of concurrent threads supported by the system as reported by std::hardware_concurrency()```? Just trying to avoid another gh issue asking for the same info.

Reply:
This is not accurate. On Linux as you said we do so. On Windows we get the number of physical cores from Windows API and use that value. Ideally on Linux we should do the same, but Linux doesn't have a unified API for it, unless we are ok to introduce a new dependency lib to do the job.

Reply:
Is it accurate (for Linux and Windows) to say it is half the number of cores?

Reply:
It is not accurate: physical cores vs logical cores. It depends on whether hyper threading is enabled or not.

Reply:
Updated the description based on off line discussion